Kajabi/sage-lib
# Add supporting site links

## Description

This PR is WIP to clean up and improve documentation:

- remove unused `use_legacy_html_code_source`
- move images into `docs/assets` instead of `docs/lib...` to test image path issues with deploy
- add links to storybook and sassdocs in header and on components main page
- test adding links to storybook for specific components.

### Screenshots

|  Screenshot  |  Image  |
|---|---|
| Header | ![Screen Shot 2021-02-26 at 9 57 10 AM](https://user-images.githubusercontent.com/17955295/109316248-69d4c780-7819-11eb-93d5-85291db210cc.png) |
| Components page | ![Screen Shot 2021-02-26 at 10 03 08 AM](https://user-images.githubusercontent.com/17955295/109316696-e1a2f200-7819-11eb-8632-3372d22aa2e2.png) |
| Element/Object preview | ![Screen Shot 2021-02-26 at 9 59 22 AM](https://user-images.githubusercontent.com/17955295/109316302-7822e380-7819-11eb-8c55-001577c19670.png) |
| Element/Object detail | ![Screen Shot 2021-02-26 at 9 59 37 AM](https://user-images.githubusercontent.com/17955295/109316318-7ce79780-7819-11eb-98ae-a36692a0adbd.png) |


## Test notes


### Steps for testing
1. Open local dev site.
2. Check links in the header that point to Storybook and Sassdocs to confirm they successfully open respective sites.
3. Check same links in the Getting Started page under Components where Sass and React are described.
4. Browse to Elements and Objects page and test where Button and Alert (respectively) show a new React icon with a link to each component's page in Storybook.

No impact on `kajabi-products`
